### PR TITLE
Women of hp article draft

### PR DESCRIPTION
**Pull request by Kinghams22**

### CODE REVIEW

File: _posts/2019-03-07-women-of-honeypot.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+Over to you, Isabel...
+
+<!--more-->
+
+### Isabel Hübner, Accounting and Finance Manager
+
+<br />
+
+<p align="center"><img alt="Isabel Hübner" src="/assets/images/debbie-l.jpg" style="width:100% !important;"></p><br>
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: wrong image

File: _posts/2019-03-07-women-of-honeypot.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+### Wiktoria Vowles, Customer Development Manager
+
+<br />
+
+<p align="center"><img alt="Wiktoria Vowles" src="/assets/images/wikta-v.jpg" style="width:100% !important;"></p><br>
+
+<br />
+
+“I was so shy when I first joined Honeypot. Not just in a professional sense, but also personally. It was difficult for me to share my ideas, to be more out there or to feel like part of a group. But something changed for me here. Being surrounded by people who challenge me and care about me injected confidence and drive; I learnt to speak my mind, to share my ideas and to not question myself so much. I learnt its ok to fail, to make mistakes. We all do. You have to make mistakes to learn. Its sounds very cliche, but I didn’t necessarily know that.
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: wikta's is the only one surrounded in quotes? (and the last paragraph)

File: _posts/2019-03-07-women-of-honeypot.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+There are many things I love about Honeypot. First is the people. I dare say there aren’t many companies like ours. The atmosphere and sense of community is incredible. It makes coming to work a real pleasure. My parents and so many people told me I would miss my university life when I started working, but that’s not the case here. I would never change my life back to what it was before. I’m really happy. The other reason I like working here is the impact we make. In the past I helped international developers to secure their visas. You get first hand experience of how a job can change your life.  I’m incredibly grateful to be part of that. 
+
+Even though I’m not 100% sure what I will do with my life professionally, I know that I want to keep facing my fears, my challenges and my flaws. If you do it and you look back to compare yourself then and now, it’s an incredible feeling. Its like you turned into a completely different person, in a good way. It pushes you forward because it makes you think, if I keep on working, what could be next?”
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: `Its` should be `It's` in the second last sentence

File: _posts/2019-03-07-women-of-honeypot.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+### Emma Hunt, People Operations Manager
+
+<br />
+
+<p align="center"><img alt="Emma Hunt" src="/assets/images/emma-h.jpg" style="width:100% !important;"></p><br>
+
+<br />
+
+“In uni, I never knew what I wanted to do. At the time, it seemed like all my friends had an exact career path in mind. All I knew was I wanted to work closely with people. It was pure coincidence that I managed to develop a career in People Ops - it’s all thanks to some amazing managers who saw the potential in me.
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: EmmaH's is just quoted at the start but not the end

File: _posts/2019-03-07-women-of-honeypot.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+Sure there are also days when I have the feeling I am not mastering my tasks and they just keep piling up: I know this will also change with new tools we will implement, new processes we will agree on and the things we will learn while testing and trying. I like the challenge, the exchange of ideas, the support everyone offers and the freedom of seeing failure as learning.
+
+Besides taking care of my clients I also reach out to potential clients. I do enjoy this sales side, too - client outreach, pitching and going to events, for example. But I don’t 100% see myself in a sales role in the future. I don’t believe sales is one of my true strengths and I want to focus on what I can do best. As I deeply care for people I want to be the social glue in a team that works close together. I see myself as driver, mediator and connector. Working rather on few topics I can focus on and put all my heart into then handling many many different tasks and staying in lose contact. My strength lies in my drive for quality and detail.
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: unsure if the `many many` in the last sentence is intentional or not, but it works either way :)

Comment (Ryman) [reply _posts/2019-03-07-women-of-honeypot.md]: `lose contact` should be `close contact` i think tho?

File: _posts/2019-03-07-women-of-honeypot.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+<br />
+
+<p align="center"><img alt="Tabea Müller" src="/assets/images/tabea-m.jpg" style="width:100% !important;"></p><br>
+
+<br />
+
+It’s very hard to convince me to do something if I don’t think it’s logical. In high school, I always loved math. When I finished high school, math was the obvious choice for me. But going into studies, I realized how utterly different mathematics at university level is, compared to school. In school it’s more about calculations and analysis, in universities it’s more about proofs and linear algebra. After my Bachelor and my first experiences of work I felt like I finally knew what I wanted. I wanted to be closer to software development, well, close to business logic rather than hardcore coding. 
+
+I joined Honeypot as a working student in Talent Management. I recently got promoted to Head of Data. I love to be part of our data team, because I think Data is so powerful. At first you may think it’s just a byproduct, but it’s so much more. You can discover patterns, support decisions, automate processes and understand behaviors and shape strategy.
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: unsure: last sentence is `a, b, c and d and e` which is a little weird

File: _posts/2019-03-07-women-of-honeypot.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+I love that I am part of shaping the future of the company.  I translate our business processes into first models which our Data Scientists then turn into fully fledged algorithms. And of course I support the team with whatever is needed. It’s been really nice to see the data team grow and I love empowering skilled people. Its rewarding to watch ideas become reality.
+
+<br />
+
+### Laura Schukkink, Country Lead the Netherlands
+
+<br />
+
+<p align="center"><img alt="aura Schukkink" src="/assets/images/laura-s.jpg" style="width:100% !important;"></p><br>
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: alt text typo

File: _posts/2019-03-07-women-of-honeypot.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+
+<br />
+
+When I was a kid I dreamt of becoming a musical star, on Wednesday’s and Friday’s I attended a musical club where I danced, act and sang. Although I loved it, it turned out my singing talent unfortunately did not exists. My colleagues can back this up after hearing me sing at a karaoke night in Prague during our Christmas party. Looking back it was clear at a very young age I should work with people and end up in some kind of lead position, during my bachelor I loved being in debate teams and I had no problem taking the lead in group projects.
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: Wednesdays and Fridays (no apostrophe)

Comment (Ryman) [reply _posts/2019-03-07-women-of-honeypot.md]: act -> acted ?

File: _posts/2019-03-07-women-of-honeypot.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+
+A year ago I started out as a Customer Success Manager for the Dutch market. I have a background in hospitality and this came in useful in this role. Talking to clients and making sure they were finding the right candidates for their teams. After being in this position for half a year, Honeypot gave me the opportunity to develop my leadership skills in the role of Country lead. It is great to feel this kind of support and I am learning and developing every day. 
+
+Being responsible for the client side, developer side but as well making sure PR, marketing, HR and events for the Dutch market are being rolled out makes it a great and divers role. Although the role is new to me and I’m doing many tasks for the first time, it is very rewarding, but also challenging. Mistakes, falling down and getting back up again is all part of it. The biggest motivation for me currently is the steep learning curve and to see the team in Amsterdam grow and develop. I really enjoy leading, coaching and guiding the team the best way I can. In the future, I would like to develop this even more to have an even bigger impact on business results.
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: divers -> diverse

File: _posts/2019-03-07-women-of-honeypot.md (hunk 8 above)
Comment (Ryman) [reply _posts/2019-03-07-women-of-honeypot.md]: I think but unsure: exists -> exist

File: _posts/2019-03-07-women-of-honeypot.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+
+### Priscilla Tenggara, B2B Marketing Manager
+
+<br />
+
+<p align="center"><img alt="Priscilla Tenggara" src="/assets/images/priscilla-t.jpg" style="width:100% !important;"></p><br>
+
+<br />
+
+I always thought I wanted the stability offered by big tech companies. I had previously interned at Facebook and loved it.  But when I received offers from for full-time roles, I had a weird, sinking feeling that doors to unknown adventures were closing. In the back of my mind, I knew I wanted something a bit different.
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: unsure what word is missing in `from for full-time`, maybe `them`?

File: _posts/2019-03-07-women-of-honeypot.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+
+When I joined Honeypot, I had zero experience working in startup environments. The adrenaline of ‘will I make it or will I not?’ is the type of rush motivating me to push forward. Working here has forced me to think outside the realms of small responsibility and to focus on the big picture. I get to work on all aspects of the marketing process from ideation, creation to execution - and I have a lot of responsibility. Its scary and requires self-focus and determination, but so far so good. And I’m learning a lot. I’m excited to see what’s next.
+
+<br />
+
+As you can see, our women are driving the company forward, leading teams and defining the vision we want to create. And yet sadly, we couldn't showcase each and every woman contributing to our succes, there are just too many to mention! As you can see below...
```
Comment (Ryman) [inline _posts/2019-03-07-women-of-honeypot.md]: succes -> success

Comment (Kinghams22) [reply _posts/2019-03-07-women-of-honeypot.md]: You're a legend Kevin :D Thanks for your help!

Comment (Kinghams22) [reply _posts/2019-03-07-women-of-honeypot.md]: Meant this for all the comments above, really helpful thanks mate

Comment (Kinghams22) [reply _posts/2019-03-07-women-of-honeypot.md]: Not surprised there are typos and mistakes though, I haven't proofread bits of it yet but I'll get on it tonight

**Pull request merged**